exchanges/kraken/kraken_test.go

```diff
@@ -1265,11 +1255,45 @@ func TestWsAddOrder(t *testing.T) {
 	}
 }
 
-func TestWsCancelOrder(t *testing.T) {
-	setupWsTests(t)
-	if err := k.wsCancelOrders([]string{"1337"}); err != nil {
-		t.Error(err)
+func mockWsCancelOrders(msg []byte, w *websocket.Conn) error {
+	var req WsCancelOrderRequest
+	if err := json.Unmarshal(msg, &req); err != nil {
+		return err
+	}
+	if req.Event != krakenWsCancelOrder {
+		return w.WriteMessage(websocket.TextMessage, []byte(fmt.Sprintf(`{"event":"subscriptionStatus","reqid":%d,"status":"ok"}`, req.RequestID)))
+	}
+	resp := WsCancelOrderResponse{
+		Event:     krakenWsCancelOrderStatus,
+		Status:    "ok",
+		RequestID: req.RequestID,
+		Count:     int64(len(req.TransactionIDs)),
+	}
+	if len(req.TransactionIDs) == 0 || strings.Contains(req.TransactionIDs[0], "FISH") { // Reject anything that smells suspicious
+		resp.Status = "error"
+		resp.ErrorMessage = "[EOrder:Unknown order]"
 	}
+	respJSON, err := json.Marshal(resp)
+	if err != nil {
+		return err
+	}
+	return w.WriteMessage(websocket.TextMessage, respJSON)
+}
+
+func TestWsCancelOrders(t *testing.T) {
+	t.Parallel()
+
+	k := testexch.MockWsInstance[Kraken](t, curryWsMockUpgrader(t, mockWsCancelOrders)) //nolint:govet // Intentional shadow to avoid future copy/paste mistakes
```

Comment:
Is there a way I've missed to set these to verbose? Not a change request, just something I want, but perhaps I've missed it.

Reply:
Nope, not currently. It's a pretty tight box.

I tend to switch on verbose in configtest.json and then just run one test.